packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -285,10 +283,8 @@ class _ZoomEnterTransition extends StatefulWidget {
 }
 
 class _ZoomEnterTransitionState extends State<_ZoomEnterTransition> with _ZoomTransitionBase {
-  // TODO(jonahwilliams): https://github.com/flutter/flutter/issues/106689
```

Comment:
Does this TODO still apply?

Reply:
From benchmarking, we don't want to use snapshotting on the web due to the single frame

Reply:
Done